### PR TITLE
[29275] LSG cleanup and update

### PR DESCRIPTION
**Pull request by ingamai**
### Changes

- “**Attribute lists**” updated (_attributes_group.lsg)
	- “attributes-key-value--value -user” removed
	- User with avatar removed (unused)
	- redundant “_attributes_key_value.lsg” removed 
- "**Notifications**" upload progress updated
- "**Work Packages - [Details Pane] - Activities**" updated .lsg
- "**Editable toolbar**" content moved to ‘In place editing’ .lsg file and updated
- "**Work packages table**" edited (checkboxes removed and list design updated)

- Unused class “**content-box**” and all occurrences removed
	- navigation link
	- .lsg and .sass files
- Unused "**choice**" classes and all occurrences removed (.lsg and .sass files)
- Unused "**Attribute tables**" deleted and code instead moved to section "Tables" with buttons
- Outdated class "**interactive-table**" removed in all .lsg files 
- All "**Action menu**" content moved to "Context menu" files; content updated
- Class "**-trailing-label**" removed from _forms.lsg
   --> **_This should later be completely replaced by class "form--label-with-check-box" and all occurences in .html and .sass files removed as well_**

- Empty files deleted
	- “**Grid table**” (.lsg and .sass)
	- “**Expandable group content**" (.lsg and .sass)
	- “**My Page**” (.lsg and .sass)
	- “**Work Packages - [Details Pane] - Relations**” (.lsg)

### CODE REVIEW

**Comment by ingamai** [comment]
See ticket: https://community.openproject.com/projects/openproject/work_packages/29275/activity

File: app/assets/stylesheets/content/_choice.sass [hunk 1]
```diff
@@ -26,33 +26,10 @@
 // See docs/COPYRIGHT.rdoc for more details.
 //++
 
-//a hack for IE, the width will be just fine for this element
-$choice-parent-relation-select-width: 375px
-
 //resolve the problem if inside are elements with float
 .detail-panel-description-content
```
Comment (HDinger) [inline app/assets/stylesheets/content/_choice.sass]: Please move the remaining two styles to other files, where they fit better. This file can then be removed.

File: app/assets/stylesheets/content/_context_menu.lsg [hunk 2]
```diff
@@ -1 +1,59 @@
 # Context menu
+
+```
+<div class="action-menu">
```
Comment (HDinger) [inline app/assets/stylesheets/content/_context_menu.lsg]: Please add the class `dropdown` here, too. Or add an entry for the dropdown menus.

**Review by HDinger — COMMENTED** [review]
Hey @ingamai ,

great job 👍 I found some missing or obsolete stuff:

* The control_colors.lsg and .sass file can be removed.
* For the modal: 
  * please add an example for the `-highlight` class
  * In one of the examples please use also a footer (as to be seen in the introductory modal)
  * For the closing icon, the class `op-modal--modal-close-button`
* In the widget-box: Please add an example with `-widget-box--arrow-multiline`
* Please have a look at the news section. It looks weird and the html structure does not match the one on the news page. I assume that we can remove some of the styles there as well..

**Pull request merged**